Cargo.toml

```diff
@@ -13,21 +13,23 @@ documentation = "https://docs.rs/bevy_iced"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-bevy_app = "0.11"
-bevy_derive = "0.11"
-bevy_ecs = "0.11"
-bevy_input = "0.11"
-bevy_math = "0.11"
-bevy_render = "0.11"
-bevy_utils = "0.11"
-bevy_window = "0.11"
+# NOTE: temp using "main" branch until a release with wgpu version matching iced
```

Comment:
Is this comment (and the one below) still relevant?

Reply:
ah, no. I'll remove them